package.json

```diff
@@ -87,7 +87,7 @@
     "camelcase": "^5.3.1",
     "dotenv": "^6.2.0",
     "extend": "~3.0.2",
-    "ibm-cloud-sdk-core": "^0.3.5",
+    "ibm-cloud-sdk-core": "github:IBM/node-sdk-core#release-candidate-v1",
```

Comment:
pretty sweet... for the RC for the SDK will you do an RC for the core to replace this?